src/pkg/lint/schema_test.go

```diff
@@ -190,6 +192,24 @@ components:
 	})
 }
 
+func TestValidatePackageSchema(t *testing.T) {
+	ZarfSchema = testutil.LoadSchema(t, "../../../zarf.schema.json")
+	setVariables := map[string]string{
+		"PACKAGE_NAME": "test-package",
+		"MY_COMP_NAME": "test-comp",
+	}
+	cwd, err := os.Getwd()
+	require.NoError(t, err)
+	err = os.Chdir(filepath.Join("testdata", "package-with-templates"))
+	require.NoError(t, err)
+	defer func() {
```

Comment:
Why are we doing this?

Reply:
Need to change directory because the function schema validation along with the entire create flow relies on being in the same directory as the zarf.yaml file. 

Doing a `defer func()` so the linter doesn't complain to me about not checking the error